p/themes/Swage/swage.scss

```diff
@@ -604,14 +604,15 @@ form {
 }
 
 #sidebar {
```

Comment:
Did you have a chance to consider the approach I introduced in `templace.css` with feature detection?

```css
@supports (scrollbar-width: thin) { }
```

It might not be needed when overriding only the `scrollbar-color` property, though. I cannot immediately think of potential problems.

Reply:
I think that that's the way to go, I've just been focused on the JavaScript today.

Reply:
Ok :-) If you are still making changes to the JavaScript, please paste your file in https://jshint.com (there are a couple of warnings) (there are also editor / CLI integrations if wanted).